pyaerocom/aeroval/setup_classes.py

```diff
@@ -121,6 +125,25 @@ def get_json_output_dirs(self, assert_exists=True):
 class ModelMapsSetup(BaseModel):
     maps_freq: Literal["hourly", "daily", "monthly", "yearly", "coarsest"] = "coarsest"
     maps_res_deg: PositiveInt = 5
+    plot_types: dict | set[str] = {CONTOUR}
+    boundaries: BoundingBox | None = None
+    map_observations_only_in_right_menu: bool = False
+    overlay_save_format: Literal["webp", "png"] = "webp"
+
+    @field_validator("plot_types")
+    def validate_plot_types(cls, v):
+        if isinstance(v, dict):
+            for m in v:
+                if not isinstance(v[m], set):
+                    v[m] = set([v[m]])
+                assert v[m] in PLOT_TYPE_OPTIONS
```

Comment:
The python compiler may remove all `assert` statements for optimized code. You want to have a real exception here.